citrination_client/models/design/target.py

```diff
@@ -5,7 +5,7 @@ class Target(object):
     """
     The optimization target for a design run. Consists of
     the name of the output column to optimize and the objective
-    (either "Max" or "Min")
+    (either "Max" or "Min", or a scalar value (such as "5.0"))
```

Comment:
found an open issue that referenced this docstring inaccuracy https://github.com/CitrineInformatics/python-citrination-client/issues/235